### PR TITLE
[FEATURE] Implement support for partial deliverables.

### LINKED ISSUE #9: [FEATURE] Add functionality to manage partial project deliverables.
**Is your feature request related to a problem? Please describe.**
Projects should have partial project deliverables which could be defined, edited and removed by team members. Students should be able to update a deliverable with the link to the finished product.

**Describe the solution you'd like**
Create a router to expose the endpoints needed to manage partial project deliverables.

**Describe alternatives you've considered**
N/A

**Provide usage examples**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by b0tw**
Fixes #9

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 const logger = require('morgan');
+const bodyParser = require('body-parser')
+const router = require('./routers')
 
 const app = express();
 
 // Setup logger.
 app.use(logger('dev'));
 
+app.use(bodyParser.json())
```
Comment (filipdutescu) [inline server/server.js]: I think it can be `express.json()`

**Review by filipdutescu — COMMENTED** [review]
Looks good. Let's see what and how it changes after @Yossarion's database PR

File: server/server.js (hunk 1 above)
Comment (b0tw) [reply server/server.js]: Yes, you are right. I'll modify

**Review by filipdutescu — COMMENTED** [review]
LGTM

**Pull request merged**